src/app/workflow/versions/versions.component.html

```diff
@@ -184,7 +184,7 @@
         >
       </th>
       <td mat-cell *matCellDef="let version">
-        <mat-icon data-cy="metrics" *ngIf="(version.metricsByPlatform | json) !== '{}'">check</mat-icon>
+        <mat-icon data-cy="metrics" *ngIf="version.metricsByPlatform && (version.metricsByPlatform | json) !== '{}'">check</mat-icon>
```

Comment:
When the `metrics` query parameter value is not specified, `metricsByPlatform` is null.

It will no longer be null with the above fix in in myworkflows.service.ts, but I figured I'd add just in case this template gets invoked via some other code path that I missed.